pkg/webui/console/store/index.js

```diff
@@ -29,7 +29,17 @@ import requestPromiseMiddleware from './middleware/request-promise-middleware'
 import logics from './middleware/logics'
 
 const composeEnhancers = (dev && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) || compose
-let middlewares = [requestPromiseMiddleware, createLogicMiddleware(logics)]
+
+const userTimingMiddleware = () => next => action => {
```

Comment:
Do we need this in master?

Reply:
Oh, this slipped through from performance debugging. Will remove.